packages/client/src/components/SafeSettings/components/Owners/ReviewRemoveSafeOwner.js

```diff
@@ -0,0 +1,69 @@
+import { useContext } from "react";
+import { useModalContext } from "contexts";
+import { useHistory } from "react-router-dom";
+import { Web3Context } from "contexts/Web3";
+import { formatAddress } from "utils";
+import OwnerCard from "./OwnerCard";
+
+const ReviewRemoveSafeOwner = ({
+  owner,
+  newThreshold,
+  safeOwners,
+  onBack,
+  treasury
+}) => {
+  const history = useHistory();
+  const { closeModal } = useModalContext();
+  const { proposeRemoveSigner, setTreasury } =
+  useContext(Web3Context);
+
+  const { address } = treasury
+
+  const renderRow = (label, value) => {
+    return (
+      <div className="columns m-0 py-1  border-light-top">
+        <label className="column px-0 has-text-grey">{label}</label>
+        <div className="column ml-2 px-0 has-text-black has-text-weight-bold">{value}</div>
+      </div>
+    );
+  };
+
+  const onSubmitClick = async () => {
+    await proposeRemoveSigner(formatAddress(owner.address), newThreshold);
+    setTreasury(address, {
+      safeOwners: safeOwners.filter((o) => o.address !== owner.address),
+    });
```

Comment:
I think we should remove this, because the signer now disappears from Settings page when we propose this.